apps/docs/docusaurus.config.js

```diff
@@ -108,16 +114,20 @@ module.exports = {
             items: [
               {
                 label: '@rx-angular/cdk',
-                to: 'docs/cdk/api/transformation-helpers',
+                to: 'docs/cdk/',
               },
               {
                 label: '@rx-angular/state',
-                to: 'docs/state/getting-started/overview',
+                to: 'docs/state/',
+              },
+              {
+               label: '@rx-angular/template',
+                to: 'docs/template/',
+              },
+              {
+                label: '@rx-angular/eslint-plugin',
+                 to: 'docs/eslint-plugin/',
               },
```

Comment:
suggestion: Sort documentation footer links alphabetically by package name.
```suggestion
              {
                label: '@rx-angular/cdk',
                to: 'docs/cdk/api/transformation-helpers',
                to: 'docs/cdk/',
              },
              {
                label: '@rx-angular/eslint-plugin',
                 to: 'docs/eslint-plugin/',
              },
              {
                label: '@rx-angular/state',
                to: 'docs/state/getting-started/overview',
                to: 'docs/state/',
              },
              {
               label: '@rx-angular/template',
                to: 'docs/template/',
              },
```